tests/nfv/vsperf_installation.pm

```diff
@@ -24,13 +24,15 @@ use utils;
 
 sub run {
     my $vsperf_repo = "https://gerrit.opnfv.org/gerrit/vswitchperf";
+    my $dpdk_repo   = "http://dpdk.org/git/dpdk";
```

Comment:
with this change - this is not just vsperf installation , right ? how about renaming module to give more clear name ? Also summary of module needs to be updated.

Reply:
I have moved it to the script that installs openvswitch and dpdk, it makes more sense there. Thanks for the suggestion.
Summary updated